### ISSUE TITLE
可以添加一个自定义CSS风格的选项吗?

### ISSUE THREAD

**Issue by ZhuoChenchen** [marktext/marktext#174]
如题, 内置一些可选风格如github等, 或提供一个手动选择css文件的选项

**Comment by shvchk** [comment]
Basic override `custom.css` loaded from the same location as `preference.md` (`~/.config/marktext/` on Linux) would be probably easy to implement and enough for most use cases.

**Comment by Jocs** [comment]
@shvchk Yes, you can rewrite (override) the css file (or .vue file) in project to custom style, and build Mark Text by yourself.

**Comment by ejm554** [comment]
Can someone clarify the status of custom themes? For example, when is it expected that a user will be able to customize paragraph spacing, etc. 

I see confusing bits of information, and I can't get a handle on where it's at. Not criticizing, mind you. Just asking for someone to make it a little less muddy for me. :-)

It sounds like *maybe* this will be possible when an plug-ins become a thing. Yet, I see the following in [marktext/docs/THEMES.md](https://github.com/marktext/marktext/blob/03afc1ec29291ed3dccc2754771a0c2c1fbb1600/docs/THEMES.md):

> MarkText currently doesn't support user-defined application themes. This feature is planned for v0.17.0.

As of this writing, the current release is 0.17.1.

**Comment by hunyun** [comment]
How to make a custom theme ???

**Comment by viocha** [comment]
+1

**Comment by DYH-Hong** [comment]
A [fork](https://github.com/jacobwhall/marktext) with actual maintenance has been created with this issue fixed.

If you can, please subscribe it and help test this feature there.

**Comment by Jocs** [comment]
Thank you for the original request! Both features asked for here have been implemented in current versions of MarkText:

1. **Built-in themes**: Preferences → Theme now offers 35 selectable themes (Dracula, Nord, Tokyo Night, Catppuccin Mocha/Latte, Solarized, Gruvbox, One Dark, Rose Pine, and many more), with automatic light/dark mode switching.

2. **Custom CSS**: The same Preferences → Theme panel has a "Custom CSS" textarea where you can paste any CSS rules; they are injected live into the editor without requiring a restart or a source build.

Closing this as implemented. If you need something beyond what these two mechanisms provide, please open a new issue with a specific description.
